scripts/superdesk-dashboard/world-clock/world-clock.js

```diff
@@ -2,43 +2,8 @@
     'use strict';
 
     angular.module('superdesk.dashboard.world-clock', [
-        'superdesk.dashboard', 'ngResource'
+        'superdesk.dashboard', 'ngResource', 'superdesk.datetime'
```

Comment:
this `ngResource` dependency should be set now on `superdesk.datetime` module instead